### ISSUE TITLE
Live caches vanish when disabling updates

### ISSUE THREAD

**Issue by rsudev** [cgeo/cgeo#5929]
When you open the (new) live map, wait for caches to load and then switch to offline, the downloaded caches are removed from the map.
They should stay, obviously.

**Comment by andrixnet** [comment]
Mapsforge standard, using offline map.
Note the zoom level and the caches.
Displayed caches are caches stored offline. (in fact they were never taken from live, but imported from GPX)
![screenshot_2016-10-25-19-12-53](https://cloud.githubusercontent.com/assets/3382777/19697100/ab94e262-9af2-11e6-861c-30a414a14993.png)

------------------------
Mapsforge beta. Just switched it on in settings.
Note the different zoom level (for same location). No zooming was performed by the user. 
Note there are no caches shown. 
Options to hide own/found/disabled  are all off.
![screenshot_2016-10-25-19-13-15](https://cloud.githubusercontent.com/assets/3382777/19697124/c632c918-9af2-11e6-9e9c-a1677632b82f.png)

**Comment by andrixnet** [comment]
Stored offline caches, as shown on mapsforge default map. Live not enabled.
![screenshot_2016-10-26-15-13-42](https://cloud.githubusercontent.com/assets/3382777/19729670/7ad8f128-9ba0-11e6-9c27-b7ec022114ca.png)

Stored caches not shown on mapsforge beta with live disabled. 
![screenshot_2016-10-26-15-14-00](https://cloud.githubusercontent.com/assets/3382777/19729688/8c131306-9ba0-11e6-9892-6398cc8e6efa.png)

I have just discovered that the meaning of "enable live" changed for mapsforge beta.

    mapsforge default - enables search and download of caches on current map viewport (brief data) while all stored caches are always shown.
    Mapsforge beta - enable live enables the displa of stored caches AND searching online for caches in viewport.

IMO the mapsforge beta behaviour is wrong.
Moved here from https://github.com/cgeo/cgeo/issues/6032#issuecomment-256331300

**Comment by Lineflyer** [comment]
I agree. It is not intended here, that disabling live mode hides saved caches.

**Comment by rsudev** [comment]
@Lineflyer PLease check if you find the current semantics fitting.
- On live map, if started with disabled live, nothing will be shown.
- On other maps, the defined content will be shown (single cache with waypoints, list)
- After enabling live once, stored and downloaded caches will be shown
- Disabling live again will keep displaying stored caches but not downloading new ones.

**Comment by kirchwitz** [comment]
I'm confused. :-)

Does the previous comment mean, if c:geo is started, the "Live map" will not show any stored caches as long as the user doesn't toggle Disable/Enable live at least once? That would be different from current behaviour (Live map always shows stored caches) which is just perfect.

Sorry, maybe I got that wrong. Just trying to understand. I'm usually always on "Disable live" when outdoors. So hopefully you may understand my confusion.

**Comment by Lineflyer** [comment]
@rsudev Correct

@kirchwitz The problem you reported is fixed in our developer version bit not yet live.

**Comment by rsudev** [comment]
@kirchwitz You are right, and this use-case I missed, as I was thinking more about enable/disable live on the map of a list or single cache, where this makes sense IMO.
For the live map as such, it seems sensible to always start with the stored caches shown.
You see, I usually never disable live on the live map, I normally just switch off mobile data instead...

**Comment by Lineflyer** [comment]
I quickly checked the "old" map:
Here it is that saved caches are initially visible, if live map is started with live disabled.
So we should stay with the same functionality also in the new map.
